spring-integration-ftp/src/main/java/org/springframework/integration/ftp/outbound/FtpMessageHandler.java

```diff
@@ -48,10 +48,15 @@ public FtpMessageHandler(FtpRemoteFileTemplate remoteFileTemplate) {
 		super(remoteFileTemplate);
 	}
 
+	@Deprecated
 	public FtpMessageHandler(RemoteFileTemplate<FTPFile> remoteFileTemplate, FileExistsMode mode) {
 		super(remoteFileTemplate, mode);
 	}
 
+	public FtpMessageHandler(FtpRemoteFileTemplate ftpRemoteFileTemplate, FileExistsMode mode) {
```

Comment:
Please, add some JavaDocs to all the new public API.

Reply:
Added